robots/hydrus/config/multi/multi_vibration_202007/Battery.yaml

```diff
@@ -0,0 +1,5 @@
+bat_info:
```

Comment:
What does `multi` stand for, and what is the difference between `multi` and `octo`?